src/actions/EngineerActions.js

```diff
@@ -137,14 +143,14 @@ export const fetchEngineer = (id) => async (dispatch) => {
         headers: {
           Authorization: `Bearer ${token}`,
           'Content-Type': 'application/json',
-          'Access-Control-Allow-Origin': `${process.env.API_URL}`,
-        },
+          'Access-Control-Allow-Origin': `${process.env.API_URL}`
+        }
       });
       const { data: cohort } = res2.data;
       console.log('cohorts', cohort);
```

Comment:
Remove log mesages.

Reply:
Create few of them and test again

Reply:
I will do that but the app should handle cases where there are no ratings.